src/content/app/tools/blast/BlastPage.scss

```diff
@@ -0,0 +1,6 @@
+@import 'src/styles/common';
+
+.blastPageContainer {
+  --blast-page-standard-padding: 0 #{$standard-gutter} #{$global-padding-bottom}
+    #{$double-standard-gutter};
+}
```

Comment:
So here's what worries me; and what I tried to express yesterday.

The element wrapped inside the newly added div has a `height: 100%` rule:

![image](https://user-images.githubusercontent.com/6834224/203741824-7fa863fa-9d73-488f-9aa7-36f34bd1c564.png)

But the parent element has no notion of a height:

![image](https://user-images.githubusercontent.com/6834224/203742949-d82b407b-55f1-434a-a56b-1e640a4c060d.png)

which means that the child's `height: 100%` becomes meaningless. If we wanted to maintain the height appropriately, we would need to set it on this parent element as well.